app/ui/runtime/stacks/index.jsx

```diff
@@ -10,11 +10,18 @@ const threadsWithStatus = (threads, status) =>
 export const RuntimeStacks = props => {
   const threads = props.machineState
     .get("threads")
-    .map((thread, index) => thread.set("name", `Stack ${index}`));
+    .map((thread, index) =>
+      thread.set("name", `Stack ${index}`).set("index", index),
+    );
 
   return (
     <Segment attached padded>
-      <Header content="Stacks" />
+      <Header textAlign="left">
+        <Header.Content>Stacks</Header.Content>
+        <Header.Subheader>
+          Click on each stack for decide your path
```

Comment:
Hmmm, this isn't really clear. I would do something like `Click on a stack to select which thread you want to execute on each step.`.